VideoLocker/src/main/java/org/edx/mobile/view/MyCourseListFragment.java

```diff
@@ -0,0 +1,257 @@
+package org.edx.mobile.view;
+
+import android.app.Activity;
+import android.databinding.DataBindingUtil;
+import android.os.Bundle;
+import android.support.v4.app.LoaderManager;
+import android.support.v4.content.Loader;
+import android.support.v4.widget.SwipeRefreshLayout;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import org.edx.mobile.R;
+import org.edx.mobile.base.BaseFragment;
+import org.edx.mobile.core.IEdxEnvironment;
+import org.edx.mobile.databinding.FragmentMyCourseListBinding;
+import org.edx.mobile.databinding.PanelFindCourseBinding;
+import org.edx.mobile.event.EnrolledInCourseEvent;
+import org.edx.mobile.exception.AuthException;
+import org.edx.mobile.http.HttpResponseStatusException;
+import org.edx.mobile.interfaces.NetworkObserver;
+import org.edx.mobile.interfaces.NetworkSubject;
+import org.edx.mobile.loader.AsyncTaskResult;
+import org.edx.mobile.loader.CoursesAsyncLoader;
+import org.edx.mobile.logger.Logger;
+import org.edx.mobile.model.api.EnrolledCoursesResponse;
+import org.edx.mobile.module.analytics.ISegment;
+import org.edx.mobile.module.prefs.LoginPrefs;
+import org.edx.mobile.util.NetworkUtil;
+import org.edx.mobile.util.ViewAnimationUtil;
+import org.edx.mobile.view.adapters.MyCourseAdapter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.inject.Inject;
+
+import de.greenrobot.event.EventBus;
+
+public class MyCourseListFragment extends BaseFragment implements NetworkObserver, LoaderManager.LoaderCallbacks<AsyncTaskResult<List<EnrolledCoursesResponse>>> {
+
+    private static final int MY_COURSE_LOADER_ID = 0x905000;
+
+    private MyCourseAdapter adapter;
+    private FragmentMyCourseListBinding binding;
+    private Logger logger = new Logger(getClass().getSimpleName());
+    private boolean refreshOnResume = false;
+
+    @Inject
+    IEdxEnvironment environment;
+
+    @Inject
+    LoginPrefs loginPrefs;
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        adapter = new MyCourseAdapter(getActivity(), environment) {
+            @Override
+            public void onItemClicked(EnrolledCoursesResponse model) {
+                MyCourseListFragment.this.environment.getRouter().showCourseDashboardTabs(getActivity(), MyCourseListFragment.this.environment.getConfig(), model, false);
```

Comment:
No need of `MyCourseListFragment.this.` here we can directly access `environment`.